packages/block-editor/src/hooks/border.js

```diff
@@ -160,15 +160,17 @@ export function BorderPanel( { clientId, name, setAttributes, settings } ) {
 		return null;
 	}
 
+	const defaultBorderControls = 
```

Comment:
Out of curiosity, have you considered updating the `core/blocks` store to be able to fall back to the experimental name when the new name is not found? In effect, in all places for the edited files, we would only need to use the non-experimental name. In addition to that, `getBlockSupport` would handle the deprecated name as needed:

https://github.com/WordPress/gutenberg/blob/351e0926138454cc97cba89e6ff68b8856169c8b/packages/blocks/src/store/selectors.js#L629-L645

Reply:
I like this idea. Ideally it'd be good if we can handle the fallback to the experimental name within `getBlockSupport` (or the store) rather than each of the block supports needing to know about the two different property names.

That said, this PR also isn't a lot of code, so if it winds up being complex to implement I don't mind the duplication too much — but it would be good to neaten up if it's feasible.

Reply:
I don't think there is a great hurry to stabilize these so I'd vote for doing it once properly for all block supports.

Reply:
Hi @gziolo.,

I am continuing with this issue and feedback from you in a new [PR](https://github.com/WordPress/gutenberg/pull/67073) I have updated the getBlockSupport function as mentioned by you. Please have a look and let you know your feedback. 

cc @andrewserong, @ndiego

Reply:
Hi @karthick-murugan 👋 

Theres been some unfortunate overlap of efforts to stabilizing block supports lately. The general consensus has been to consolidate stabilization of block supports around the approach pioneered in https://github.com/WordPress/gutenberg/pull/63401.

I already have a PR close to landing for the border block supports in https://github.com/WordPress/gutenberg/pull/66918. The hold up in landing that was I was also iterating there so that the same stabilizing functions and filters also took care of both `__experimentalDefaultControls` and `__experimentalSkipSerialization`. 

At this stage, I should have the finalized PRs up today or tomorrow for review. Rather than scatter the stabilization of block supports across various utils and selectors. I strongly believe we should have a consistent location for handling them.

I'll drop a comment over on https://github.com/WordPress/gutenberg/pull/67073 as well with the hope to get us all on the same page again.